routes/product.js

```diff
@@ -0,0 +1,22 @@
+const express = require("express");
+const { upload } = require("../helpers/helper");
+const productController = require("../controllers/product");
+const Validate = require("../controllers/validator");
+
+const router = express.Router();
+
+router.post(
+  "/",
+  upload.array("images"),
+  Validate.Productvalidate("insertProduct"),
+  productController.insertProduct
+);
+router.put(
+  "/:id",
```

Comment:
why there is no option to update image? and in discussion I also mentioned there should be option to mark one of them as featured...can you please implement that?

Reply:
done